packages/ui/src/elements/ShimmerEffect/index.tsx

```diff
@@ -6,21 +6,29 @@ import './index.scss'
 
 export type ShimmerEffectProps = {
   readonly animationDelay?: string
+  readonly className?: string
+  readonly disableInlineStyles?: boolean
   readonly height?: number | string
   readonly width?: number | string
 }
 
 export const ShimmerEffect: React.FC<ShimmerEffectProps> = ({
   animationDelay = '0ms',
+  className,
+  disableInlineStyles = false,
   height = '60px',
   width = '100%',
 }) => {
   return (
     <div
-      className="shimmer-effect"
+      className={['shimmer-effect', className].filter(Boolean).join(' ')}
       style={{
-        height: typeof height === 'number' ? `${height}px` : height,
-        width: typeof width === 'number' ? `${width}px` : width,
+        height: disableInlineStyles
+          ? undefined
```

Comment:
Do you need this double ternary here? `disableInlineStyles` has a default value of `false` from in the props.

Reply:
Updated with:
```js
height: !disableInlineStyles && (typeof height === 'number' ? `${height}px` : height),
width: !disableInlineStyles && (typeof width === 'number' ? `${width}px` : width),
```